packages/ddp/livedata_tests.js

```diff
@@ -601,29 +611,109 @@ if (Meteor.isClient) {
       }
     ];})());
 
-    testAsyncMulti("livedata - publish multiple cursors", [
+  testAsyncMulti("livedata - publisher errors with onStopped callback", (function () {
```

Comment:
copy and pasting the whole test is not really ideal (though the ugly thing we do where we make tests that run in multiple guises isn't super great either) but better than no test